### PR TITLE
fix: remove locales from Microsoft links

### PR DESCRIPTION
**Pull request by nschonni**
Used https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack to cleanup the MS links to drop the locales. They do this because they redirect to the correct locale based on the user, so it's easier for i18n, because links don't need to change in the localization repos

### CODE REVIEW

**Review by Elchi3 — COMMENTED** [review]
The diff is not clean. There seem to be unrelated changes to removing "en-us" from Microsoft links.

**Comment by nschonni** [comment]
The other diffs are from replacing the random NBSP characters when I had to review the file changes

File: files/en-us/web/api/websockets_api/writing_websocket_server/index.md [hunk 1]
```diff
@@ -15,15 +15,15 @@ This server conforms to [RFC 6455](https://datatracker.ietf.org/doc/html/rfc6455
...
+WebSockets communicate over a [TCP (Transmission Control Protocol)](https://en.wikipedia.org/wiki/Transmission_Control_Protocol) connection. Luckily, C# has a [TcpListener](https://msdn.microsoft.com/library/system.net.sockets.tcplistener.aspx) class which does as the name suggests. It is in the *System.Net.Sockets* namespace.
 
 > **Note:** It is a good idea to include the namespace with the `using` keyword in order to write less. It allows usage of a namespace's classes without typing the full namespace every time.
 
 ### TcpListener
 
 Constructor:
 
-```cpp
+```cs
```
Comment (Elchi3) [inline files/en-us/web/api/websockets_api/writing_websocket_server/index.md:26]: This is not an nbsp change

Comment (nschonni) [reply files/en-us/web/api/websockets_api/writing_websocket_server/index.md:26]: right, I noticed that when I was reviewing the changes. The document is talking about C# examples and code, but the code blocks were incorrect

Comment (Elchi3) [reply files/en-us/web/api/websockets_api/writing_websocket_server/index.md:26]: Can you separate this out?

Comment (nschonni) [reply files/en-us/web/api/websockets_api/writing_websocket_server/index.md:26]: Not easily, but feel free to close if you think this touches too much

Comment (estelle) [reply files/en-us/web/api/websockets_api/writing_websocket_server/index.md:26]: if this is the only issues, we can revert this inline by making a comment commit.

Comment (estelle) [inline files/en-us/web/api/websockets_api/writing_websocket_server/index.md:26]:
```suggestion
```cpp
```

Comment (teoli2003) [inline files/en-us/web/api/websockets_api/writing_websocket_server/index.md:26]: I think we can fix this PR with this commit (and follow-up this in a separate PR). Am I wrong?
```suggestion
```cpp
```

**Review by rebloor — APPROVED** [review]
I've reviewed the two web extension pages affected. Noted the replacement of spaces, but no other unexpected changes. The revised links worked as expected.

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**